lib/options.go

```diff
@@ -235,7 +250,7 @@ type Options struct {
 
 	// Specify TLS versions and cipher suites, and present client certificates.
 	TLSCipherSuites *TLSCipherSuites `json:"tlsCipherSuites" envconfig:"K6_TLS_CIPHER_SUITES"`
-	TLSVersion      *TLSVersions     `json:"tlsVersion" envconfig:"K6_TLS_VERSION"`
+	TLSVersion      *TLSVersions     `json:"tlsVersion" ignored:"true"`
```

Comment:
FWIW I think we could fix the envconfig issue with a [custom decoder](https://github.com/kelseyhightower/envconfig#custom-decoders) and keep the `K6_TLS_VERSION` env var, but since it's not documented in the k6 docs, and making it work is tricky in general because of #883, I'm OK with ignoring it.

Reply:
We can't keep delaying the envconfig removal, so let's ignore this for now and fix it properly everywhere soon.